src/paint.js

```diff
@@ -304,6 +304,8 @@ function setupPaint({ $, qlik }) {
           .range(colors);
 
         gridSize = Math.floor((width - margin.left - margin.right) / gridDivider);
+        console.log(gridSize);
```

Comment:
remove logging and it looks like this is the only warning for no-console, so making that an error instead of warning aswell would be nice